resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -85,24 +85,25 @@ function createListElement(date, type, text, id, author) {
   return liElement;
 }
 
-/**
- * Fetches the blobstore-serve to sends its response as an array buffer to the Adobe DC View
- */
+ //Adobe Preview configurations for getRevieweeResume function
  const previewConfig={
   'showLeftHandPanel':true,
   'showPageControls':true,
   'showDownloadPDF': false,
-  'showAnnotationTools': true,
   'showPrintPDF': false,
   'enableAnnotationsAPI': true,
   'includePDFAnnotations': true
```

Comment:
Nit: Change all the keys in this object to not be string types.

```
const previewConfig={
  showLeftHandPanel: true,	
  showPageControls: true,	 
  showDownloadPDF: false,	 
  showPrintPDF: false,	
  enableAnnotationsAPI: true,	 
  includePDFAnnotations: true,
}
```

You also don't need showAnnotationTools field in this object because you'll set it in `getRevieweeResume()`.
```